cms/src/util/slug.spec.ts

```diff
@@ -22,8 +22,17 @@ describe("Slug", () => {
         db.docs.put({ ...mockEnglishContentDto, slug: "sample-title" } as ContentDto);
         const title = "Sample Title";
 
-        const generatedSlug = await Slug.generate(title);
+        const generatedSlug = await Slug.generate(title, "different-id");
 
         expect(generatedSlug).toMatch(/sample-title-[0-9]([0-9])*/);
     });
+
+    it("should not make the title for the same content unique", async () => {
```

Comment:
Is the test title correct? Should it not read something like "should not make the SLUG for the same content [document] complete"? (Corrections in uppercase and additional clarifying text in [ ])

Reply:
Renamed the variable to `documentId` as well to be a bit more generic as it could be other things than content with slugs in the future.